src/lib/consensus/proof_of_stake_fuzzer.ml

```diff
@@ -152,7 +152,7 @@ module Vrf_distribution = struct
     in
     let rec extend_proposal_chain acc_chain slot =
       let potential_proposals = find_potential_proposals [] 0 slot in
-      if List.length potential_proposals = 0 then acc_chain
+      if potential_proposals = [] then acc_chain
```

Comment:
Isn't `=` monomorphised to `int` in core now?

Reply:
I indeed had to fix one instance of this somewhere else. Good catch.